src/auto/functions.rs

```diff
@@ -300,20 +312,42 @@ pub fn networking_init() {
 //    unsafe { TODO: call ffi::g_pollable_source_new_full() }
 //}
 
-//#[cfg(any(feature = "v2_34", feature = "dox"))]
-//pub fn pollable_stream_read<'a, P: IsA</*Ignored*/InputStream>, Q: Into<Option<&'a Cancellable>>>(stream: &P, buffer: &[u8], blocking: bool, cancellable: Q) -> Result<isize, Error> {
-//    unsafe { TODO: call ffi::g_pollable_stream_read() }
-//}
+#[cfg(any(feature = "v2_34", feature = "dox"))]
+pub fn pollable_stream_read<'a, P: IsA<InputStream>, Q: Into<Option<&'a Cancellable>>>(stream: &P, buffer: &[u8], blocking: bool, cancellable: Q) -> Result<isize, Error> {
```

Comment:
This should not be in functions, but in InputStream (and the other one in OutputStream). Also calling this with blocking=false on a non-GPollableInputStream is undefined behaviour, thus unsafe. I would just not expose this at all, it's C convenience stuff with footguns

Reply:
Also the buffer needs to be writable

Reply:
Ok I ignore this function.

Reply:
Maybe ignore `pollable_stream_write` too? PollableStreams not generated.

Reply:
Yes, I meant both